frontend/test/metabase/scenarios/admin/settings/sso/ldap.cy.spec.js

```diff
@@ -34,3 +53,7 @@ function findToggleByLabel(text) {
 function findInputByLabel(text) {
   return findByLabel(text, "input");
 }
```

Comment:
Ideally, we should have labels connected to the form elements naturally, or at least by using the `aria-labelledby`. That would allow us to use `cy.findByLabelText` and it wouldn't matter if you're searching for `input`, toggle, or  a `checkbox`.

Reply:
I know, but I'm not sure if we should go this far and refactor settings UI in this PR as well. WDYT about creating a separate ticket with the issue?

Reply:
Oh, absolutely. We should keep PRs as small (and as focused) as possible.